go/node/cmd/main.go

```diff
@@ -1,13 +1,19 @@
 package main
 
 import (
+	"time"
+
 	"github.com/obscuronet/go-obscuro/go/node"
 )
 
 func main() {
 	cliConfig := ParseConfigCLI()
 	// todo (#1618) - allow for multiple operation (start, stop, status)
 
+	// we already validated the config in ParseConfigCLI, so we can safely ignore the error here
+	batchInterval, _ := time.ParseDuration(cliConfig.batchInterval)
+	rollupInterval, _ := time.ParseDuration(cliConfig.rollupInterval)
```

Comment:
What if the node validated the time parsing and this mechanism only took strings and simple types ?